docs/operations.md

```diff
@@ -123,6 +167,128 @@ stays in the database:
 
 **Default**: `30` (days)
 
+## Database option
```

Comment:
at the moment this is its own section on the same level as 'Scavenging' itself and also 'Backup and restore', it should probably be part of the scavenging section

there are already some dabase options described above